public/rnnoise/rnnoise_sdk.mjs

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
All these new files under public folder are krisp/rnnoise sdk files correct? If so, I think we need to put them in an assets folder, organized by sdk name and version each file to allow cache-busting on the consumer side. We should also add a doc (before release) to recommend copying the files over and serve them in their web app. It can be done using post install scripts.
For folder structure, maybe something like
```
lib/assets/<sdk name>/<filename>-<version>.<extension>
```

Reply:
Thanks @charliesantos, I will address this before release.

Reply:
now moved the asset files to `assets/<sdk-name>/<version>/files`